Analyzer/src/MakeMiniTree.cc

```diff
@@ -10,110 +10,195 @@ MakeMiniTree::MakeMiniTree()
+            "PrimaryVertexFilter",
+            "BadPFMuonFilter",
+            "EcalDeadCellTriggerPrimitiveFilter",
+            "HBHEIsoNoiseFilter",
+            "HBHENoiseFilter",
         };
-        if( runtype != "MC" ) {
-            variables = {
-            "FinalLumi",
-            "filetag",
-            "NGoodJets_pt30",
-            "NGoodBJets_pt30",
-            "Mbl",
-            "NVtx",
-            "HT_trigger_pt30",
-            "HT",
-            "NGoodElectrons",
-            "NGoodMuons",
-            "MET",
-            };
+
+        if( runtype == "MC" )
+        {
+            variables.insert("Weight");
+            variables.insert("madHT");
+            variables.insert("GenElectrons");
+            variables.insert("GenMuons");
+            variables.insert("GenTaus");
+            variables.insert("GenMET");
+            variables.insert("GenMETPhi");
+            variables.insert("GenParticles");
+            variables.insert("GenParticles_PdgId");
+            variables.insert("GenParticles_ParentId");
+            variables.insert("GenParticles_ParentIdx");
+            variables.insert("GenParticles_Status");
+            variables.insert("ScaleWeights");
+            variables.insert("PSweights");
+            variables.insert("PDFweights");
+            variables.insert("puWeight");
+            variables.insert("puSysUp");
+            variables.insert("puSysDown");
+            variables.insert("NonPrefiringProb");
+            variables.insert("NonPrefiringProbUp");
+            variables.insert("NonPrefiringProbDown");
+            variables.insert("Jets_origIndex");
+            variables.insert("JetsJECup_origIndex");
+            variables.insert("JetsJECdown_origIndex");
+            variables.insert("JetsJERup_origIndex");
+            variables.insert("JetsJERdown_origIndex");
+            variables.insert("Jets_jerFactor");
+            variables.insert("Jets_jecUnc");
+            variables.insert("JetsJECup_jerFactor");
+            variables.insert("JetsJECdown_jerFactor");
+            variables.insert("Jets_jerFactorUp");
+            variables.insert("Jets_jerFactorDown");
+            variables.insert("JetIDJECdown");
+            variables.insert("JetIDJECup");
+            variables.insert("JetIDJERdown");
+            variables.insert("JetIDJERup");
+            variables.insert("JetsAK8_origIndex");
+            variables.insert("JetsAK8JECup_origIndex");
+            variables.insert("JetsAK8JECdown_origIndex");
+            variables.insert("JetsAK8JERup_origIndex");
+            variables.insert("JetsAK8JERdown_origIndex");
+            variables.insert("JetsAK8_jerFactor");
+            variables.insert("JetsAK8_jecUnc");
+            variables.insert("JetsAK8JECup_jerFactor");
+            variables.insert("JetsAK8JECdown_jerFactor");
+            variables.insert("JetsAK8_jerFactorUp");
+            variables.insert("JetsAK8_jerFactorDown");
```

Comment:
Are you throwing away all variables in the nTuples that we don't use? There are some datasets with tracking information that takes up a lot of room that we want to remove. In fact these tracking variables take up more room than any other variable including the PF candidates.

Reply:
Trying my best to exclude what we do not need (save for a few variables I saw that we might want/have to look at e.g. nvtx and rho pileup parameter. I started the other way around, with too few variables (from what I saw at a glance from the basic modules), and just built up the list to the current state until I could run `AnalyzeDoubleDisCo` on the skimmed file.

Reply:
That sounds like a good strategy. I was just pointing out mostly that we will save alot of space for some of the large MC samples by throwing out some of these variables.